backend/api.py

```diff
@@ -1,12 +1,54 @@
 # create our little application :)
-from flask import Flask
+import configparser
+from flask import Flask, request
+import tweepy
 
 app = Flask(__name__)
 
+def are_you_the_keymaster():
+    config = configparser.ConfigParser()
+    config.read('keys.ini')
+    return config
 
-@app.route('/opinion')
+
+def twitter_handler(message):
+    twconf = are_you_the_keymaster()['twitter']
+    auth = tweepy.OAuthHandler(twconf['conkey'], twconf['consec'])
+    auth.set_access_token(twconf['atok'], twconf['atoksec'])
+    api = tweepy.API(auth)
+    return str(api.update_status(message))
+    #return str(api.verify_credentials())
+
+
+def choose_handler():
+    # TODO: add other handlers
+    def handler(message):
+        print("Sending message", str(message))
+        return twitter_handler
+    return handler
+
+
+def validate_message(message):
+    print("Testing message", message)
+    return message != ''
+
+
+def handle_message(message):
+    if not validate_message(message):
+        return 'Bad message!', 400
+
+    handler = choose_handler()
```

Comment:
I actually don't lol. It's a little unpythonic to ask for permission instead of asking for forgiveness, but I figured we'd be changing it as we add handlers anyway-- we have to decide what happens when a message is valid for one handler but not others, etc. (I say just drop it but we can decide)